### DISCUSSION TITLE
Timeout for Entirety of WebDriverIO Test Runner Instance

### DISCUSSION THREAD

**Discussion by samuelfreiberg** [webdriverio/webdriverio#12670] [Q&A]
Is there a way to create a timeout for the entire WebDriverIO test runner instance? Not specific to Wait() commands, but the entire test runner instance. 

For example, if I want to set a timeout for 7 minutes from the first instance WebDriverIO test runner starts.

**Comment by christian-bromann** [comment]
Hey @samuelfreiberg 👋 no, such a timeout configuration does not exist. Please raise a feature request and provide some detailed description about your use case and why you think this would be useful.

Thank you!

**Reply by samuelfreiberg** [reply]
https://github.com/webdriverio/webdriverio/issues/12678
